Caesar_cipher/gonvers/main.py

```diff
@@ -0,0 +1,45 @@
+class CaesarCypher():
+    chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    chars_len = len(chars)
+    def __init__(self, key=0):
```

Comment:
between methods of class should be empty line
https://www.flake8rules.com/rules/E301.html